lib/package.json

```diff
@@ -44,13 +44,15 @@
   },
   "dependencies": {
     "babel-runtime": "^6.26.0",
+    "luxon": "^1.0.0",
```

Comment:
hmm, shouldn't `luxon` be a dev dependency? And also be located in `externalDependencies` along with moment and date-fns?

Reply:
Jeez, yes, I totally forgot about that. I just ran `yarn add luxon` locally which adds that as a dependency automatically (like `npm install --save`. Yes, that should be changed.

Reply:
no worries ;)
would you like to submit a PR?

Reply:
Also, it would be great to document luxon localization on our docs page

Reply:
I just noticed that the performance with Luxon is not really that great compared to, for example, the DateFns util. I will investigate this later this week.